### ISSUE TITLE
Feature Request: AOS Responsive (Enable animations based on screen size)

### ISSUE THREAD

**Issue by COD3BREAK3R** [michalsnik/aos#811]
As the title says, it would be great if AOS had breakpoints that allowed you to apply animations based on the window size.

**Comment by souravm84** [comment]
Yes, This is very important. fade-right,fade-left causing screen overflow in small screens like mobile.
